crowdloans/projects/kusama.json

```diff
@@ -316,5 +316,28 @@
     "paraid": "2107",
     "name": "KICO",
     "icon": "https://auction-us-east-2.s3.us-east-2.amazonaws.com/icons/chains/kico.svg"
+  },
+  {
+    "paraid": "2114",
+    "name": "Turing",
+    "token": "TUR",
+    "description": "Automation Blockchain for Kusama",
+    "overview": "Turing is an automation blockchain for Defi and recurring payments across protocols. Turing aims to be the best-in-class automation utility across protocols, offering on-chain automation not possible today.",
+    "website": "https://oak.tech/turing/home/",
+    "icon": "https://s3.amazonaws.com/docs.oak.tech/assets/img/turing.png",
```

Comment:
could you add your icon to our repo?(https://github.com/parallel-finance/auction-toolkit/tree/main/icons/chains) And use `https://auction-us-east-2.s3.us-east-2.amazonaws.com/icons/chains/<iconName>` link for the icon